content/en/docs/tutorials/quickstart.md

```diff
@@ -100,18 +103,18 @@ When opening the DevContainer for the first time, a manual reload of the dapr ex
 <details>
 <summary>Please expand for information on troubleshooting</summary>
 
-> If Visual Studio Code fails to directly clone your repository you can also use a workaround:
+> If Visual Studio Code [fails to directly clone your repository](https://github.com/microsoft/vscode-dev-containers/issues/1585)  you can also use a workaround:
 >
-> 1. clone the repo locally using your favorite Git tooling
-> 1. Start Visual Studio Code
-> 1. select `Open Folder` from the `File` menu
-> 1. open the root of the cloned repo
-> 1. a popup appears on the lower left side of Visual Studio Code
-> 1. click on `Reopen in Container`
-> 1. wait for the container to be set up
+> 1. Clone the repo locally using your favorite Git tooling
+> 2. Start Visual Studio Code
+> 3. Select `Open Folder` from the `File` menu
+> 4. Open the root of the cloned repo
+> 5. A popup appears on the lower left side of Visual Studio Code
+> 6. Click on `Reopen in Container`
+> 7. Wait for the container to be set up
```

Comment:
using 1. all the time is better for adaptations since github is adding the correct number automatically.

Reply:
Reverted